assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -46,6 +46,10 @@ function useUploadMedia({ media, pagingNum, mediaType, fetchMedia, setMedia }) {
 
   const uploadMedia = useCallback(
     async (files, { onLocalFile, onUploadedFile, onUploadFailure } = {}) => {
+      // If there are no files passed, don't try to upload.
```

Comment:
Shouldn't this check be here - https://github.com/google/web-stories-wp/blob/fb620eeea3c8791ba57f08c42f5f9c78a13c8955/assets/src/edit-story/components/canvas/useUploadWithPreview.js#L78-L87

Reply:
Thought about it, however, it looks like `uploadMedia` should never start the upload process if the `files` is empty -- added it directly to the root instead then. Makes sense to have it in `uploadMedia` directly rather than adding the check to everywhere using it.

Or is there a case when it should run when `files` is empty?